commands/roll.js

```diff
@@ -116,16 +64,16 @@ module.exports = {
 		else game = await ctx.bot.getGame(ctx, 'myz');
 
 		// Creates the roll.
-		let roll = new YZRoll(game, ctx.author, name);
+		let roll = new YZRoll(game, ctx.author, name, lang);
 
 		// Year Zero Roll Regular Expression.
-		const yzRollRegex = /^((\d{1,2}[dbsgna])|([bsgna]\d{1,2})|(d(6|8|10|12))|([abcd])+)+$/i;
+		const yzRollRegex = /^((\d{1,2}[dbsfgnaw])|([bsfgna]\d{1,2})|([dw](6|8|10|12))|([abcd])+)+$/i;
```

Comment:
I am not supporting a change of the regex based on language alternatives for "d".
Please, revert back to original this line and the other.

Reply:
Alright, will do.

Reply:
Reverted that commit